### PR TITLE
GH-23: Add PR template

### LINKED ISSUE #23: Add pull request template
If we need it.

See also apache/arrow's one: https://github.com/apache/arrow/blob/main/.github/pull_request_template.md

### PR DESCRIPTION
**Pull request by zeroshade**
Fixes #23

### CODE REVIEW

File: .github/pull_request_template.md
Comment (kou) [inline .github/pull_request_template.md:1]: How about using raw text instead of comment text?

It seems that most developers don't remove comment in pull request template.
In apache/arrow, we remove comments in our merge script https://github.com/apache/arrow/blob/0c3206772051c52fe2e648623daaca2c4e71cb72/dev/merge_arrow_pr.py#L603-L604 but we don't use merge script in apache/arrow-go. So our commit history have many this comments...

If we use raw text instead of comment text, we can notice it before we use the merge button. We can remove needless texts before we merge.

**Review by kou — COMMENTED** [review]
We need `Fixes #XXX` in PR description because we don't use a merge script.

**Comment by zeroshade** [comment]
@kou how about a simple template with just the headings? I think we can give enough faith in developers to understand the sections without worrying about them having to remember to delete sample text?

**Review by kou — APPROVED** [review]
+1

Let's try the approach!

**Pull request merged**
